solr/contrib/prometheus-exporter/conf/solr-exporter-config.xml

```diff
@@ -1053,6 +1053,51 @@
           </str>
         </arr>
       </lst>
+      <lst name="request">
+        <lst name="query">
+          <str name="path">/admin/zookeeper/status</str>
```

Comment:
For Exporter to be able to query this endpoint with SolrJ, SOLR-15384 is required for SolrJ. This is already committed.